src/BabelBot.Worker/appsettings.Development.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  }
+}
```

Comment:
I think this one is still left in the old path too 😅

I'm struggling with the same issue too (although I'm using the launchSettings.json) - actually getting dotnet user-secrets working might be the next thing I'll work on, since those would be the canonical way to solve that problem.

Reply:
Ugh. Let me clone the repo again so I can make sure to fix it this time.
I'm all for the canonical way :D

Reply:
Actually getting those working seems to be trivial, I'll push the patch needed to enable after this merge :)

Reply:
Done. Hopefully 😄

Reply:
Done. Hopefully 😄